src/strings/ops.c

```diff
@@ -644,7 +642,34 @@ MVMint64 MVM_string_equal_at_ignore_case(MVMThreadContext *tc, MVMString *haysta
     }
     return 1;
 }
+MVMint64 MVM_string_index_ignore_case(MVMThreadContext *tc, MVMString *haystack, MVMString *needle, MVMint64 start) {
+    size_t index           = (size_t)start;
+    MVMStringIndex hgraphs = MVM_string_graphs(tc, haystack), ngraphs = MVM_string_graphs(tc, needle);
```

Comment:
Should not do these assignments until after the `MVM_string_check_arg` calls below, otherwise may dereference a NULL pointer or read out of bounds.